client/python/openlineage/client/transport/http.py

```diff
@@ -160,6 +162,9 @@ def set_adapter(self, adapter: HTTPAdapter) -> None:
             self.session.mount(self.url, adapter)
 
     def emit(self, event: Event) -> Response:
+        # If anyone overrides debuglevel manually, we can potentially leak any secrets to logs.
+        # Override this setting to make sure it does not happen.
+        http_client.HTTPConnection.debuglevel = 0
```

Comment:
Is it a global change? Maybe we should set it back to 1 after we emit the event?

Reply:
Changed.